MCprep_addon/materials/prep.py

```diff
@@ -291,14 +295,21 @@ def execute(self, context):
 		if self.optimizeScene and engine == 'CYCLES':
 			bpy.ops.mcprep.optimize_scene()
 
+		has_no_prep = count_no_prep > 0
+		has_lib_skipped = count_lib_skipped > 0
+		has_mat = count > 0
+		
+		_info = {}
+		_info[f"modified {count}"] = has_mat
+		_info[f"skipped {count_lib_skipped} linked"] = has_lib_skipped
+		_info[f"founded {count_no_prep} no prep"] = has_no_prep
+		
+		mat_info = ", ".join(k for k, v in _info.items() if v).capitalize()
```

Comment:
I'm not the biggest fan of this, as it's not really clear what this does. I would make this separate if statements and just push elements into some `info` list. Dictionaries don't really make sense here. In addition, `_info` doesn't make sense here, as it's not a private member of a class.

```suggestion
		has_no_prep = count_no_prep > 0
		has_lib_skipped = count_lib_skipped > 0
		has_mat = count > 0
		
		info = []
		if has_mat:
			info.append("modified {count}")
		if has_lib_skip:
			info.append("skipped {count_lib_skipped}")
		if has_no_prep:
			info.append("{count_no_prep}" not prepped")		
		mat_info = ", ".join(x for x in str).capitalize()
```

As an aside, perhaps we should lump `count_lib_skipped` and `count_no_prep` into one count, since they're both skipping materials.

Reply:
I'm fine with grouping these together. Only other consideration is to try and make info reports rather short, since they fill out the bottom bar in the UI and are fleeting and non-blocking.

I would personally prefer keeping the short message, and group them all into one category for the purpose of the info report: f"Modified {count} materials, {skip_count} skipped" and leave it at that.

I am, however, totally fine with printing out more of this to the console. I also support @StandingPadAnimations simplified string cat (just be sure to put `f` ahead of some of those strings using vars)